Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
Looks like we got a mix of tab and space whitespacing, let's keep it all tabs

Reply:
Cleaned up!